client/src/modules/stock/aggregated_consumption/templates/quantity_remain.tmpl.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents">
+  {{ row.entity.old_quantity - row.entity.quantity_consumed - row.entity.quantity_lost }}
```

Comment:
What about the quantity received during the month?  Is that part of "old_quantity"?

Reply:
You are right

Reply:
@jniles  it's run well
![image](https://user-images.githubusercontent.com/8035588/121889005-9224fb00-cd10-11eb-94d5-c2cbe4c0a318.png)

Reply:
@jniles  it's run well
![image](https://user-images.githubusercontent.com/8035588/121889005-9224fb00-cd10-11eb-94d5-c2cbe4c0a318.png)

Reply:
Thanks @lomamech !